setupext.py

```diff
@@ -677,15 +688,28 @@ def do_custom_build(self, env):
 
             cc = get_ccompiler()
             cc.initialize()  # Get msbuild in the %PATH% of cc.spawn.
+            # Freetype 2.10.0+ support static builds.
+            msbuild_config = (
+                "Release Static"
+                if version.parse(LOCAL_FREETYPE_VERSION) >= version.parse("2.10.0")
+                else "Release"
+            )
+
             cc.spawn(["msbuild", str(sln_path),
                       "/t:Clean;Build",
-                      f"/p:Configuration=Release;Platform={msbuild_platform}"])
+                      f"/p:Configuration={msbuild_config};Platform={msbuild_platform}"])
             # Move to the corresponding Unix build path.
             (src_path / "objs" / ".libs").mkdir()
             # Be robust against change of FreeType version.
-            lib_path, = (src_path / "objs" / vc / msbuild_platform).glob(
-                "freetype*.lib")
-            shutil.copy2(lib_path, src_path / "objs/.libs/libfreetype.lib")
+            lib_paths = Path(src_path / "objs").rglob('freetype*.lib')
+            libfreetype_file = None
+            for lib_path in lib_paths:
+                # check if freetype.lib in required platform directory
+                if msbuild_platform in lib_path.resolve().as_uri():
+                    libfreetype_file = lib_path
+                    continue
+            print("Copying {} to {}".format(lib_path.resolve(), src_path / "objs/.libs/libfreetype.lib"))
+            shutil.copy2(libfreetype_file, src_path / "objs/.libs/libfreetype.lib")
```

Comment:
The print and the actual copy appear that they can do different things?

Reply:
Fixed!